docs/ops/comparison/NotEqual_1.md

```diff
@@ -13,7 +24,8 @@
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
     * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in
+      <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
```

Comment:
also pdpd mode is missed. Please make sure that implementation in nGraph reference contains it